boards/arm/tiva/tm4c1294-launchpad/src/tm4c1294-launchpad.h

```diff
@@ -77,6 +77,9 @@
 #  undef HAVE_I2CTOOL
 #endif
 
+/* Define the procfs mounting point */
+#define TIVA_PROCFS_MOUNTPOINT "/proc"
```

Comment:
Please add empty line after comment line

Reply:
Done in [997e60a](https://github.com/apache/nuttx/pull/8152/commits/997e60aaddec0694f4ca171eeeae0adc57b66ab3)